app/services/calculators/property_calculator.rb

```diff
@@ -60,7 +65,7 @@ def calculate_main_home_disregard(property, submission_date)
 
       # TODO: Remove this side effect
       def save!(property, result)
-        property.update!(result.to_h.except(:smod_applied))
+        property.update!(result.to_h.except(:smod_allowance, :property))
```

Comment:
With properties now returned via the PersonCapitalSubtotals, I think we might be nearly at the point where we can stop saving this stuff to the database entirely. Nice!